### ISSUE TITLE
Unknown prop `sharer` on <span> tag

### ISSUE THREAD

**Issue by balmasich** [olahol/react-social#43]
drop warning that sharer is unknown prop
```
import ReactSocial from 'react-social';
const { FacebookButton, TwitterButton } = ReactSocial;
<FacebookButton
            element={'span'}
            url={url}
            sharer
            className={s.fb}
            appId={sharing.facebook.app_id}
            windowOptions={['status=0', 'toolbar=0', 'width=480', 'height=350']}
          >
```
warning.js:36 Warning: Unknown prop `sharer` on <span> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop

**Comment by savovs** [comment]
Having the same issue, it's a bit annoying to see the big red warning in the console.

**Comment by olahol** [comment]
Thanks, fixed this in `1.6.2`.
